prep-html-syntax/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Hi Jae, be sure to include a `<head></head>` element before your `<body></body>` element.